assets/audio/sequences/seq_0.prg.seq

```diff
@@ -13793,6 +13798,10 @@ CHAN_5EAB:
 /* 0x5EAF [0xF3 0x06               ] */ rbeqz       CHAN_5EB7
 /* 0x5EB1 [0xF1 0x01               ] */ allocnotelist 1
 /* 0x5EB3 [0xD1 0x02               ] */ notealloc   2
+#if OOT_PAL_N64
+/* 0x5EC6 [0xCC 0x00               ] */ ldi         0
+/* 0x5EC8 [0x76                    ] */ stio        IO_PORT_6
```

Comment:
```suggestion
                                        ldi         0
                                        stio        IO_PORT_6
```
The comments in the PAL-only code can probably just be omitted since it's unclear what they mean when the comments in the rest of the file only apply to non-PAL versions?

Reply:
Yeah this is an oversight, I removed the offset comments but kept the data bytes